cilium/cmd/endpoint_config.go

```diff
@@ -84,7 +84,7 @@ func configEndpoint(cmd *cobra.Command, args []string) {
 		if err != nil {
 			Fatalf("Cannot parse option %s: %s", opts[k], err)
 		}
-		modifiedOptsCfg.Options[name] = fmt.Sprintf("%d", value)
+		modifiedOptsCfg.Options[name] = fmt.Sprintf("%t", value)
```

Comment:
`ParseOption()` now returns an integer, not a bool - I think that your local cache is out of date, perhaps?

Reply:
Confirmed by CI failure:
```
12:37:43 cilium/cmd/endpoint_config.go:87: Sprintf format %t has arg value of wrong type int
```